### PR TITLE
Button symbols to text

### PR DESCRIPTION
**Pull request by Samcraig77**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -65,14 +65,28 @@ h2 {
...
 
 /**BUTTONS**/
-.reward-btn {
-    background-color: #9E4770;
+.reward-btns {
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    width: inherit;
+
```
Comment (Raehale) [inline index.css:73]: Is an ending bracket missing here or is that just github weirdness?

File: index.css [hunk 2]
```diff
@@ -65,14 +65,28 @@ h2 {
...
 /**BUTTONS**/
-.reward-btn {
-    background-color: #9E4770;
+.reward-btns {
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    width: inherit;
+    
+    div {
```
Comment (Raehale) [inline index.css:74]: This is being applied to every div, if we make a div in the future we don't want to have half width, that could be a problem, maybe make a class like width-50 or smthin?

**Review by Raehale — CHANGES_REQUESTED** [review]
Great Job! Just a couple small things!

File: index.css [hunk 3]
```diff
@@ -122,7 +137,7 @@ h2 {
 
 
 /***DESKTOP***/
-@media screen and (min-width: 1200px) {
+@media screen and (min-width: 1250px) {
```
Comment (Raehale) [inline index.css:140]: This is gonna increase the size the screen has to be to be in desktop mode. Is that what you want?

File: index.css (hunk 1 above)
Comment (Samcraig77) [reply index.css:73]: There is, it's all nested inside the .reward-btns for scoping and making sure none of my changes break out of that div. I do wish github showed that change...

File: index.css (hunk 2 above)
Comment (Samcraig77) [reply index.css:74]: That's a good idea, I'm make the change!

File: index.css (hunk 3 above)
Comment (Samcraig77) [reply index.css:140]: Yes, just by 50px so it doesn't clash with the .actions bubble. I tested on two screens, but let me know if it's throwing any issues for your screens!

**Pull request merged**